.github/workflows/test_fx.yml

```diff
@@ -29,9 +29,9 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        pip install .
+        pip install .[tests]
         pip install git+https://github.com/huggingface/transformers.git
     - name: Test with unittest
       working-directory: tests
       run: |
-        python -m unittest discover -s fx -p 'test_*.py'
+        python -m pytest fx/optimization/test_transformations.py --exitfirst
```

Comment:
No we need to be able to also discover other files, for instance when quantization will be merged, we will need to discover those tests as well.